api/v2.0/swagger.yaml

```diff
@@ -8734,6 +8734,9 @@ definitions:
       skip_audit_log_database:
         $ref: '#/definitions/BoolConfigItem'
         description: Whether skip the audit log in database
+      scanner_skip_update_pulltime:
+        $ref: '#/definitions/BoolConfigItem'
+        description: Whether skip update the pull time for scanner
```

Comment:
Whether to skip the pull time for updating the scanner

Reply:
Update